pkg/make_rpm.py

```diff
@@ -298,20 +408,55 @@ def main(argv):
                            'environment variable')
   parser.add_argument('--debug', action='store_true', default=False,
                       help='Print debug messages.')
-  parser.add_argument('files', nargs='*')
+
+  ex_grp = parser.add_argument_group('EXPERIMENTAL',
+                                     'Experimental options (used by pkg/experimental/rpm.bzl')
+  ex_grp.add_argument('--install_script',
+                      help='Installer script')
+  ex_grp.add_argument('--file_list',
+                      help='File containing a list of files to include with rpm spec %files -f')
+  ex_grp.add_argument('--preamble',
+                      help='File containing the RPM Preamble')
+  ex_grp.add_argument('--description',
+                      help='File containing the RPM %description text')
+  ex_grp.add_argument('--pre_scriptlet',
+                      help='File containing the RPM %pre scriptlet, if to be substituted')
+  ex_grp.add_argument('--post_scriptlet',
+                      help='File containing the RPM %post scriptlet, if to be substituted')
+  ex_grp.add_argument('--preun_scriptlet',
+                      help='File containing the RPM %preun scriptlet, if to be substituted')
+  ex_grp.add_argument('--postun_scriptlet',
+                      help='File containing the RPM %postun scriptlet, if to be substituted')
+  ex_grp.add_argument('files', nargs='*')
 
   options = parser.parse_args(argv or ())
 
   try:
-    builder = RpmBuilder(options.name, options.version, options.release,
-                         options.arch, options.rpmbuild, options.source_date_epoch,
-                         debug=options.debug)
+    builder = RpmBuilder(options.name,
+                         options.version, options.release,
+                         options.arch, options.rpmbuild,
+                         source_date_epoch=options.source_date_epoch,
+                         debug=options.debug,
+                         # Additional scriptlet path arguments set when being
+                         # used by experimental/rpm.bzl
+                         pre_scriptlet_path=options.pre_scriptlet,
+                         post_scriptlet_path=options.post_scriptlet,
+                         preun_scriptlet_path=options.preun_scriptlet,
+                         postun_scriptlet_path=options.postun_scriptlet)
     builder.AddFiles(options.files)
-    return builder.Build(options.spec_file, options.out_file)
+    return builder.Build(options.spec_file, options.out_file,
+                         # Additional arguments set when being used by
```

Comment:
I don't see a need to say experimental here. If we put them in the main make_rpm, they are part of the interface. OTOH, they are a secret part, so I don't feel bad about breaking people who depend on internals. 
But... we intend these to be permanent features, so why not just have them as args, with no mention of experimental?

Reply:
I buy it.  I'll clean up the comments.